src/modules/Painter/tools/blackcrayon.ts

```diff
@@ -0,0 +1,16 @@
+import Tool from '@/modules/Tool';
+import {ActionType} from '@/constants';
+import blackline from './blackline';
+
+export default Tool.extend(blackline, {
+  [ActionType.MouseDown]: (
+    ctx,
+    _,
+    __,
+    unit,
+  ) => {
```

Comment:
How about get parameters as object?

Reply:
i was trying to save even the cost of creating a literal object,
but when I think about it now, I don't think there will be a big difference in performance.

thanks for good feedback. i'll edit 👍